resources/fetching/fetchers/azure/batch_fetcher.go

```diff
@@ -57,29 +57,58 @@ func (f *AzureBatchAssetFetcher) Fetch(ctx context.Context, cMetadata fetching.C
 		}
 
 		if len(assets) == 0 {
+			// Hack backported into 8.11:
+			// Get the first subscription (should only be one), get the sub id and name and populate a mock resource
+			// with an empty array.
+			subId, subName := func() (string, string) {
+				for subId, subName := range f.provider.GetSubscriptions() {
+					return subId, subName
+				}
+				return "", ""
```

Comment:
This might create problems in the UI.
Since this should be used to create unique IDs.
Maybe if we cannot pull anything we should just skip? Sending empty strings might break some things in the UI. We need to verify that.

Reply:
This should never happen, it would mean we have no subscriptions, so we can't scan anything anyway. Added a sanity check anyway.